src/v/storage/tests/log_segment_appender_test.cc

```diff
@@ -20,18 +20,31 @@
 #include <seastar/core/gate.hh>
 
 #include <fmt/format.h>
+#include <string_view>
 
 using namespace storage; // NOLINT
 
+namespace {
+
+  ss::file open_file(std::string_view filename) {
+      return ss::open_file_dma(
```

Comment:
if this indention is being added by your editor, I recommend fixing your editor to run clang-format automatically. otherwise there will be a lot of unnecessary reformatting patches.

Reply:
Will do. And I did end up folding the formatting patches into the originals.